README.md

```diff
@@ -37,7 +37,8 @@ A "chef ingredient" is the core package itself, or products or add-on components
 - `reconfigure` - Performs the `ctl reconfigure` command for the package.
 
 #### Properties
-- `package_name`: (name attribute) The name of the package. Should correspond to the published package names (chef-server-core, opscode-manage, etc).
+- `product_name`: (name attribute) The product name. See the `#product_matrix` method in `libraries/helpers.rb` for a list of valid product names. For example, `chef-server`, `analytics`, `delivery`, `manage`, etc.
+- `package_name`: The name of the package in the repository. Should correspond to the published package names (`chef-server-core`, `opscode-manage`, etc).
```

Comment:
My thought was flexibility for transitioning, or in the event that a new package is created that isn't in the matrix (yet), without us having to cut a new chef-ingredient cookbook to update that.